examples/AcousticWave2D_codegen.py

```diff
@@ -78,14 +78,21 @@ def damp_boundary(damp):
         self.dm = dm
 
     def Forward(self):
-        fw = ForwardOperator(self.m, self.src, self.damp, self.rec, self.u, time_order=self.t_order, spc_order=self.s_order, cache_blocking=True, profile=True, auto_tune=True)
+        fw = ForwardOperator(self.m, self.src, self.damp, self.rec, self.u, time_order=self.t_order, spc_order=self.s_order, cache_blocking=True, profile=True)
         fw.apply()
-        return (self.rec.data, self.u.data)
+        return self.rec.data, self.u.data
+
+    # Forward with auto tuning enabled
+    def Forward_auto_tune(self):
```

Comment:
OK, a) travis doesn't run benchmark, because benchmark is just a scratch area for developers, not a detailed feature test.
And b) travis doesn't have Isat, so we won't be able to auto-test your stuff in the test suite anyway. So I would say simply having a flag in `benchmark.py`, and some documentation somewhere that the feature exists, should be enough for anybody who wants to test or run this.